tools/downloader/common.py

```diff
@@ -71,6 +71,7 @@
     'place_recognition',
     'question_answering',
     'salient_object_detection',
+    'scene_text_recognition',
```

Comment:
what the difference between optical character recognition and scene text recognition? I mean why do you need additional model type?

Reply:
These tasks are a bit different. Optical character recognition is about recognizing straight text on a simple background, whereas scene text recognition (which could be considered as a special form of OCR) aims to recognize text of different (maybe curved) forms in a complex background. 
![image](https://user-images.githubusercontent.com/32310433/115345345-57429300-a1b7-11eb-84d9-371601b612b3.png)

Reply:
then why text-recognition-0012/0013 have `optical_character_recognition` as task name?
https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/text-recognition-0012/model.yml#L18
(it is automatically generated files, so we can not modify them directly)
It may be confusing for users to see models in different categories in downloader... (I am not against have correct task name in documentation, but for downloading it can be overabundant info)







overabundant |  
-- | --

Reply:
Ok, so what will we do? Change the name for this task or change domain for text-recognition-0012\13?